draft-ietf-quic-qpack.md

```diff
@@ -1119,7 +1119,7 @@ HTTP_QPACK_DECODER_STREAM_ERROR (0x202):
 
 # Security Considerations
 
-TBD.
+TBD.  Also see Section 7.1 of [RFC7541].
```

Comment:
This allows the reference to Section 7.1 in Literal Header Field With Name Reference section be replaced with an internal reference to this section.